nio/responses.py

```diff
@@ -182,17 +188,18 @@
 def verify(schema, error_class, pass_arguments=True):
     def decorator(f):
         @wraps(f)
-        def wrapper(cls, parsed_dict, *args, **kwargs):
+        def wrapper(cls, parsed_dict: Dict | None, *args, **kwargs):
```

Comment:
FYI the `type | type` syntax was introduced in py3.10, if you want to keep 3.8+ support, you'll need to use typing.Union (or in this case Optional[]) still